app/controllers/transactions_controller.rb

```diff
@@ -7,6 +7,12 @@ def create
     @transaction.save
   end
 
+  def create_transfer
+    @transfer = Transfer.new(transaction_params)
+    @transaction = Transaction.new(transaction_params)
+    @out_transaction, @inc_transaction  = @transfer.save(@transaction)
```

Comment:
@Gataniel don't return transactions on save. User something like `@transfer.in_transaction` to get transactions after save. `save` method should return `true`/`false`